### PR TITLE
[BACKLOG-32582] Add jpivot to marketplace

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon please review.

### CODE REVIEW

**Comment by davidmsantos90** [comment]
@pentaho/millenniumfalcon pull request updated. please review

File: marketplace.xml [hunk 1]
```diff
@@ -698,6 +698,69 @@
...
+      <name>Analysis</name>
+      <parent>
+        <name>Apps</name>
+      </parent>
+    </category>
+
+    <license>CPL-1.0</license>
+
+    <installation_notes>
+      This plugin requires a jar (JavaServer Faces API) that was removed form the Pentaho Server.
```
Comment (dcleao) [inline marketplace.xml]: I'd suggest something like:
«
To function, this plugin requires that a JAR dependency is installed in the Pentaho Server - the JavaServer Faces API. This dependency has known security vulnerabilities, which are described in the security issue CVE-2008-1285. Installing this dependency in the Pentaho Server compromises its security.

To install the JAR dependency, download it from 'https://mvnrepository.com/artifact/javax.faces/jsf-api/1.1', place it in the 'tomcat/webapps/pentaho/WEB-INF/lib' folder and finally restart the Pentaho Server.
»

File: marketplace.xml [hunk 2]
```diff
@@ -698,6 +698,69 @@
...
   </market_entry>
 
+
+  <market_entry>
+    <id>pentaho-jpivot-plugin</id>
+    <type>Platform</type>
+
+    <name>Pentaho Platform JPivot Plugin</name>
+    <description>
+      This is a port of the JPivot functionality that existed in Pentaho Platform 4.8 into a stand alone plugin.
```
Comment (dcleao) [inline marketplace.xml]: I'd suggest something like:
«
The legacy and deprecated JPivot plugin.
This plugin is no longer supported. Furthermore, it is vulnerable to known security vulnerabilities.
Hitachi Vantara advises strongly against installing this plugin.
Install this plugin at your own risk.
See the installation instructions, for more information.
»

**Comment by davidmsantos90** [comment]
@pentaho/millenniumfalcon pull request updated. please review

@pamval do you think the description and installation notes are appropriate to JPivot Plugin?

**Comment by bennychow** [comment]
I think the current plugin description sounds a little extreme.  It would probably be better to actually describe what this plugin does:

JPivot is a JSP custom tag library that renders an OLAP table and let users perform typical OLAP navigations like slice and dice, drill down and roll up. It uses Mondrian as its OLAP Server. JPivot also supports XMLA datasource access.

Source: http://jpivot.sourceforge.net/

And then add a warning like:  This plugin is deprecated due to security vulnerabilities.

**Comment by dcleao** [comment]
@bennychow I agree with you, in a way. However, my reasoning was that we do not want to advertise the plugin — if you don't already know what it does... you don't need it, you don't wanna use it...
Anyway, I think your text would be ok for the description text, but I would leave the additional security remarks on the installation notes section.

**Comment by pmalves** [comment]
I agree with Benny's description. The current one is waaaay too aggressive.
A "use at your own risk" should be more than enough

**Comment by davidmsantos90** [comment]
@pentaho/millenniumfalcon please review.

@pmalves @bennychow changed the description to explain the plugin and included a warning saying it isn't supported anymore.

**Pull request merged**